spec/models/team_spec.rb

```diff
@@ -102,6 +102,13 @@
     expect { team.members << user }.to change { ActionMailer::Base.deliveries.length }.by(1)
   end
 
+  it 'should not notify newly added team members when team is a single' do
```

Comment:
You could change the „single“ wording into „created by event“ here as well to avoid confusion.

Reply:
:+1: